src/components/ContentSidebar/ActivityFeed/activity-feed/__tests__/ActivityFeed-test.js

```diff
@@ -135,6 +135,39 @@ describe('components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed', ()
         expect(wrapper.find('[name="addApproval"]').length).toEqual(0);
     });
 
+    test('should set scrollTop to be the scrollHeight if feedContainer ref is set', () => {
+        const wrapper = shallow(<ActivityFeed currentUser={currentUser} />);
+        const instance = wrapper.instance();
+        instance.feedContainer = {
+            scrollTop: 0,
+            scrollHeight: 100,
+        };
+
+        instance.componentDidMount();
```

Comment:
Instead of calling component instance methods, can we interact just through props and DOM events? That makes it possible to refactor the component without breaking the test.

Reply:
Will address this in a bit